recipes/ibdseq/ibdseq

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Wraps ibdseq.jar
+#
+# This is copied from https://github.com/chapmanb/gatk
+#
+set -o pipefail
```

Comment:
There is a python version of this wrapper that we slightly prefer over the bash one. Maybe you can use this for the next release.